### PR TITLE
Create_contact_form

### PR DESCRIPTION
**Pull request by Munsa1**
Hello😎
In the following pull request, I have done the following and met the requirements

- Created the contact form section of my portfolio
- Added checks to ensure validation for the Email text field
- Restricted name text field characters to 30 as requested
- Restricted text area character limit to 500 characters as requested
- Added placeholder text to all parameters
- Ensured user cannot submit the form if one or more of the fields are not filled with the required data
- Applied collection of data using formspree service

Your feedback will be highly appreciated😜

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -92,56 +93,77 @@ <h4 class="card-heading">Multi-Post Stories Gain+Glory</h4>
...
 
 </main>
+
+<footer class="footing">
+<h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+<div>
+<form class="form-container user-form" action="https://formspree.io/f/xbjqyrpj" method="POST">  
+    <input type="text" class="user-info" placeholder="Full name" maxlength="30" required>
+    <input type="email" class="user-info" placeholder="Email address" required>
+    <textarea  class="user-comments" cols="30" rows="10" placeholder="Enter text here" maxlength="500"  required></textarea>
```
Comment (Davidosky007) [inline index.html]: I suggest message input's type equals textarea as stated in the [Project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)

File: index.html [hunk 2]
```diff
@@ -92,56 +93,77 @@ <h4 class="card-heading">Multi-Post Stories Gain+Glory</h4>
...
+                    </ul>
+                </div>   
+        </li>
     </ul>
     </section>
 
 </main>
+
+<footer class="footing">
+<h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```
Comment (Davidosky007) [inline index.html:150]: stick to the design templates in Figma as much as possible, this includes the font, colours, images and spacing (margin/padding). use the right font family for this text.

File: index.html (hunk 1 above)
Comment (Davidosky007) [inline index.html]: sticks to the design templates in Figma as much as possible, this includes the font, colours, images and spacing (margin/padding).  use the right font family and colour for the palceholders.

File: index.html [hunk 3]
```diff
@@ -92,56 +93,77 @@ <h4 class="card-heading">Multi-Post Stories Gain+Glory</h4>
...
+<div>
+<form class="form-container user-form" action="https://formspree.io/f/xbjqyrpj" method="POST">  
+    <input type="text" class="user-info" placeholder="Full name" maxlength="30" required>
+    <input type="email" class="user-info" placeholder="Email address" required>
+    <textarea  class="user-comments" cols="30" rows="10" placeholder="Enter text here" maxlength="500"  required></textarea>
+    <input class="form-button green-button" type="submit" value="Get in touch">
```
Comment (Davidosky007) [inline index.html]: * When the form is submitted this error occurs, please fix this
Screenshot (19)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @ntzwilly @Munsa1, 


Good job so far! Love your commit messages and PR summary keep it up 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I suggest you fix the error below in the comments that happen when the submit button is clicked. follow these links for hints [Link1](https://www.w3schools.com/tags/att_input_required.asp) [Link2](https://www.w3schools.com/tags/att_input_name.asp)
* Please check the comments below and make the necessary changes 👍
# _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Munsa1) [reply index.html]: Thank you, I have fixed this

**Comment by Munsa1** [comment]
Thank you for the feedback👍 I have revisited my code and made all the required changes

File: index.html (hunk 3 above)
Comment (Munsa1) [reply index.html]: Thank you, its been handled

File: index.html (hunk 1 above)
Comment (Munsa1) [reply index.html]: I was having trouble with the font style being a linter error but its fixed now

File: index.html (hunk 2 above)
Comment (Munsa1) [reply index.html:150]: Thank you for that as well it has been handled

File: index.html (hunk 1 above)
Comment (Dipeshtwis) [reply index.html]: This issue was successfully addressed.

File: index.html (hunk 2 above)
Comment (Dipeshtwis) [reply index.html:150]: This issue was successfully addressed. :+1:

File: index.html (hunk 1 above)
Comment (Dipeshtwis) [reply index.html]: This issue was successfully addressed. :+1:

File: index.html (hunk 3 above)
Comment (Dipeshtwis) [reply index.html]: Able to send the detail. :+1:

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @Munsa1, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**